spec/support/acts_as_tenant.rb

```diff
@@ -9,10 +9,12 @@ def with_tenant(tenant, &proc)
     context "with tenant #{tenant.inspect}" do |*params|
       before(:each) do
         ActsAsTenant.current_tenant = send(tenant)
+        @old_host = Capybara.app_host
```

Comment:
Yes, Then I think we might need a similar function for controllers/features.

Because this function: https://github.com/Coursemology/coursemology2/blob/master/app/controllers/concerns/application_multitenancy_concern.rb#L6

always changes the tenant even if you declared <code>with_tenant</code> inside controller/feature test.